ai_genomics/getters/gtr.py

```diff
@@ -0,0 +1,24 @@
+from ai_genomics import PROJECT_DIR, logger
+import pandas as pd
+import re
+
+GTR_INPUTS_DIR = PROJECT_DIR / "inputs/data/gtr"
+
+
+def get_ai_genomics_gtr_data(data_type: str) -> pd.DataFrame:
+    """Get AI and Genomics GtR data
+
+    Args:
+        data_type: "projects" or "publications"
```

Comment:
maybe it would make more sense to call this `entity_type` or something similar? When talking about data types I would normally think of str, int, etc.

Reply:
Also a nice to have would be some handling to raise an error if another option (or typo) is passed in.